tests/integration/__tests__/cache.js

```diff
@@ -445,7 +445,7 @@ describe("--cache option", () => {
       );
     });
 
-    it("doesn't re-format when timestamp has been updated", async () => {
+    it("re-format when timestamp has been updated", async () => {
```

Comment:
@sosukesuzuki `file-entry-cache` v10 consider file changed if `mtime` changed even `useChecksum: true`.

https://github.com/jaredwray/cacheable/blob/e2e035296810b704b9a6e06a2628bdfa7ed1bfa0/packages/file-entry-cache/src/index.ts#L265
https://github.com/jaredwray/cacheable/blob/e2e035296810b704b9a6e06a2628bdfa7ed1bfa0/packages/file-entry-cache/src/index.ts#L299

in v9 when `useChecksum: true` it doesn't count `mtime` change, see `_getMetaForFileUsingCheckSum` and `_getMetaForFileUsingMtimeAndSize` in https://unpkg.com/browse/file-entry-cache@9.1.0/cache.js

I think it's fine to change the behavior.

Reply:
FYI this broke caching in CI workflows. See #17278